docsite/docs/Docs/fal-cli/pure-python-models.md

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 8
+---
+
+# Making pure Python dbt models
+
+In this guide we will build a dbt model with a Python script using fal.
+
+This guide is inspired by Andy Reagan's post where fal is mentioned, [Non-SQL in your DBT pipeline](https://andyreagan.medium.com/non-sql-in-your-dbt-pipeline-c7cef2091619). Which in turn is inspired by Claire Carroll's post on the dbt Discourse, [Representing non-SQL models in a dbt DAG](https://discourse.getdbt.com/t/representing-non-sql-models-in-a-dbt-dag/2083).
+
+<!-- TODO: Why would we do this? -->
+We love dbt, but it only runs SQL, some stuff can only (or much more easily) be achieved in other programming languages. In this case, we are going to use Python.
+
+## Context
+
+We are going to expand the jaffle shop classic example of dbt. We will add a life time value prediction to the shop costumers.
+
+## Make dbt _know_ about the model
+
+The first thing we want to do is to actually create a dbt model. 
+
+```sql
+{{ config(materialized='ephemeral') }}
+/*
+This table is generated by fal.
```

Comment:
This comment is confusing. What does `this` mean? Which table are we talking about?

Also, do we need to specify the `fal flow` requirement here in this comment? Shouldn't it be somewhere more visible?